### DISCUSSION TITLE
不更新go-cqhttp就用不了怎么办

### DISCUSSION THREAD

**Discussion by DaBoSAMA** [Tsuk1ko/CQ-picfinder-robot#270] [问题求助]
go-cqhttp的旧版本出现了一个问题，导致无法上线QQ，而作者表示已经在最新版解决了 这个问题，目前只有更新最新版才能解决问题。
但是更新最新版本之后bot就用不了了，这就很难受。请问有什么解决方案吗
go-cqhttp作者回复此bug的地址：[https://github.com/Mrs4s/go-cqhttp/issues/1227](url)

**Comment by Tsuk1ko** [comment]
怎么用不了，v1.0.0-beta8-fix2 bot 没问题啊

**Reply by DaBoSAMA** [reply]
”请暂时不要使用 go-cqhttp v1.0.0-beta8 版本，此版本为开发者前瞻测试版，存在许多问题”
我以为这个版本用不了了

**Reply by Tsuk1ko** [reply]
噢，当时 fix1 版本是有 bug 的所以我就这么写了，后续修复了那就没什么关系了
